### PR TITLE
CVE-2013-6622 and CVE-2016-1673

### PR DESCRIPTION
**Pull request by bam9523**
# CVE-2013-6622

CVE involves a use-after-free bug resulting from incorrect load delay timing of a media element. Elements were able to send a load event after being destroyed, creating the vulnerability.

# CVE-2016-1673

CVE involves a browser-based XSS vulnerability resulting from lack of adherence to the Same Origin Policy. Requests were able to bypass checks determining whether request navigation should be allowed when they were sent into a frame in a new document.

### CODE REVIEW

File: cves/CVE-2013-6622.yml [hunk 1]
```diff
@@ -72,10 +98,15 @@ discovered:
...
-  answer: 
-  date: 
-  automated: 
-  google: 
+  answer: |
+    The vulnerability was found by a user on the issue tracking system, who
+    appears to have a record of finding bugs (they celebrate their "10th bug
+    report" in the report). However, although they did document the stack trace
+    and unit test used, they did not document how they came across the
+    vulnerability.
```
Comment (Ransom1993) [inline cves/CVE-2013-6622.yml]: Should provide user name of the user who found this bug

File: cves/CVE-2013-6622.yml [hunk 2]
```diff
@@ -15,7 +15,22 @@ description_instructions: |
...
   keep too.
-description: 
+description: |
+  The vulnerability was a "use-after-free" bug in the fork of WebKit used by
+  Chrome. The engine provides the ability to move elements between documents,
+  and frees the memory used by the element in the first document before moving
+  the element to the second document. It was found that, with a certain timing,
+  the freed memory used by the element in the first document could be loaded
+  after it was destroyed, creating a potential denial of service.
+
```
Comment (danielroach9) [inline cves/CVE-2013-6622.yml]: +4 upvotes. I wonder how easy/hard it must have been to get the timing right to expose this vulnerability.

File: cves/CVE-2013-6622.yml [hunk 3]
```diff
@@ -106,7 +140,7 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer: 
+  answer: Not any major events were seen during the vulnerability.
```
Comment (danielroach9) [inline cves/CVE-2013-6622.yml]: Should read: "No major events were seen during the vulnerability."

File: cves/CVE-2013-6622.yml (hunk 2 above)
Comment (danielroach9) [reply cves/CVE-2013-6622.yml]: Don't know if it's only visible on my end but accidentally highlighted a line. My bad lol

File: cves/CVE-2016-1673.yml [hunk 4]
```diff
@@ -32,7 +36,9 @@ fixes_vcc_instructions: |
 fixes:
 - :commit: f93a0e557dd97cc83d8b55953c2f57c5e2dfe07b
   :note: ''
-vccs: []
+vccs:
+  - commit: cff1a5e1c110ad79007bd7bda87cfde6045cc2af
+    note: It appears they introduced the vulnerability while fixing other bugs.
```
Comment (danielroach9) [inline cves/CVE-2016-1673.yml]: Giving some information about what bugs that were trying to fix would be helpful here.

File: cves/CVE-2013-6622.yml (hunk 3 above)
Comment (bam9523) [reply cves/CVE-2013-6622.yml]: This strange wording is due to the fact that the word "No" is reserved in YAML syntax, resolving to `false`; I was concerned that the CI would reject my submission if I used it. I'll make the suggested change to see if that actually occurs.

Comment (bam9523) [reply cves/CVE-2013-6622.yml]: It appears my concerns were unfounded; the build passed.

File: cves/CVE-2016-1673.yml (hunk 4 above)
Comment (bam9523) [reply cves/CVE-2016-1673.yml]: Hopefully I was able to explain it to a satisfactory degree. The bug was apparently an OOPIF-handling error ("Out-of-process iframe") where navigation was not correctly dispatched to a remote frame. I felt a full explanation would be too jargony, so I distilled it to the parts relating to the newly-introduced vulnerability.

File: cves/CVE-2013-6622.yml (hunk 1 above)
Comment (bam9523) [reply cves/CVE-2013-6622.yml]: Addressed; the bug reporter's name is now included.

**Pull request merged**